arquillian-chameleon-deployments/arquillian-chameleon-deployment-api/src/main/java/org/arquillian/container/chameleon/deployment/api/DeploymentParameters.java

```diff
@@ -0,0 +1,23 @@
+package org.arquillian.container.chameleon.deployment.api;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface DeploymentParameters {
```

Comment:
I was wondering, can this be part of "deployment annotation" instead of living next to it?

For example 

```java
@DeploymentParameters(testable = false)
@File("src/test/resources/arquillian-example-helloworld.war")
```

would become this:

```java
@File(value = "src/test/resources/arquillian-example-helloworld.war", parameters = @DeploymentParameters(testable = false))
```

just something to consider.

Reply:
`DeploymentParameters` is a common annotation that can be used in any automatic deployment implementations based on annotations.  So at the very end we can add as parameter, but other implementator could follow another way. So at the end I decided to do it in this way because I think that they are two different things, one is to configure the deployment and the other is to configure how archive is generated. But no strong opinion here we can change if we all agree.

Reply:
I would keep it the same way as implemented

Reply:
Fine with me